### PR TITLE
Chore: Update package lock to fix CI failures

### PR DESCRIPTION
**Pull request by david-szabo97**
Bump package lock file

### CODE REVIEW

**Comment by gziolo** [comment]
Changes look good, can you provide more details why it is necessary before we merge it?

**Comment by david-szabo97** [comment]
> Changes look good, can you provide more details why it is necessary before we merge it?

We can't merge any of the PRs because of Static Analysis test failing. 
image

**Review by ntsekouras — APPROVED** [review]

**Comment by gziolo** [comment]
Thanks, merged!

**Pull request merged**
